modules/swagger-jaxrs2/src/test/java/io/swagger/jaxrs2/annotations/parameters/ParametersTest.java

```diff
@@ -53,6 +55,16 @@ public void testSimpleCallback() {
                 "          description: the generated id\n" +
                 "          format: id\n" +
                 "          readOnly: true\n" +
+                "      - name: arrayParameter\n" +
+                "        in: query\n" +
+                "        required: true\n" +
+                "        content:\n" +
+                "          '*/*':\n" +
+                "            schema:\n" +
+                "              type: int\n" +
+                "              description: the generated id\n" +
+                "              format: id\n" +
+                "              readOnly: true\n" +
```

Comment:
what should be taking precedence (array or content or schema)? did you discuss this with @webron?

Reply:
#2416 Suggested by Ron in this PR.

Reply:
@ralphdoe What do you mean? I couldn't find any reference there about the question above

Reply:
I meant this PR: https://github.com/swagger-api/swagger-core/pull/2333.
webron 28 days ago Owner
We should specify this, schema and content are mutually exclusive, and describe the behavior if more than one is specified. I'd suggest content > array > schema. @ralphdoe - what's the current behavior of the reader?